apple/internal/apple_framework_import.bzl

```diff
@@ -82,13 +82,18 @@ def _swiftmodule_for_cpu(swiftmodule_files, cpu):
     return module
 
 def _classify_framework_imports(config_vars, framework_imports):
-    """Classify a list of framework files into bundling, header, or module_map."""
+    """Classify a list of framework files into bundling, header, module_map, or static archive."""
 
     bundling_imports = []
     header_imports = []
     module_map_imports = []
+    static_archive_imports = []
     for file in framework_imports:
         file_short_path = file.short_path
+        if file_short_path.endswith(".a"):
+            # TODO: Handle cases where static archive filenames don't end with .a
```

Comment:
I haven't yet to seen one. Probably there isn't one out there, but it's a valid case because technically you can specify any file (this is controlled by the root Info.plist's `LibraryPath` value of each inner library).

Reply:
a static framework xcframework would be this case I think? like these https://github.com/keith/StaticInternalSwiftSyntaxParser/releases

In this case the file is the same name as the framework though, which we could probably use for this, even if it doesn't always exist when something with the same name does it should be the binary.

Reply:
I was thinking about the non-framework case. For example:

```
ios_static_xcframework.xcframework/Info.plist
ios_static_xcframework.xcframework/ios-arm64/Headers/shared.h
ios_static_xcframework.xcframework/ios-arm64/ios_static_xcframework_custom_name
ios_static_xcframework.xcframework/ios-arm64_x86_64-simulator/Headers/shared.h
ios_static_xcframework.xcframework/ios-arm64_x86_64-simulator/ios_static_xcframework_custom_name
```

Reply:
Anyway, let's only handle these two cases now:
1. same name as the framework
2. ends with ".a"